coins/src/adapters/tokenMapping.json

```diff
@@ -2938,6 +2962,21 @@
       "decimals": "18",
       "symbol": "ZBTC",
       "to": "coingecko#wrapped-bitcoin"
+    },
+    "0xe04d21d999faedf1e72ade6629e20a11a1ed14fa": {
+      "decimals": "18",
+      "symbol": "SolvBTC.m",
+      "to": "coingecko#solv-btc"
+    },
+    "0xb88a54ebbda8edbc1c2816ace1dc2b7c6715972d": {
+      "decimals": "18",
+      "symbol": "bcLP-stBTC-WBTC",
+      "to": "coingecko#wrapped-bitcoin"
+    },
+    "0xb750f79cf4768597f4d05d8009fcc7cee2704824": {
+      "decimals": "18",
+      "symbol": "Stable-LP",
```

Comment:
`0xb750f79cf4768597f4d05d8009fcc7cee2704824` is stBTC-WBTC Stable LP from Macaron Swap (https://info.macaron.xyz/#/pool/0xb750f79cf4768597f4d05d8009fcc7cee2704824) 


**About the StableSwap on Macaron**

> The StableSwap is an implementation of Curve Finance’s AMM on Macaron. It adds linear invariant constant sum curve (x+y=k) on top of the constant product formula (x*y=k) to keep prices more equal as long as the liquidity pool is not extremely unbalanced. As a result, since StableSwaps are restricted to similarly priced assets, impermanent loss is not as much of a concern (except in extreme depeg cases) and the slippage is lower than normal AMM which just uses the constant product formula.


https://macaron.gitbook.io/macaron-bitlayers-native-dex/stableswap